diagnostic.md

```diff
@@ -16,13 +16,14 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/Gabrakadabra/wdi/diagnostics/fundamentals-diagnostic
+~/wdi/diagnostics/fundamentals-diagnostic
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+Github is an online server that connects my computer to the web and allows me to make/edit files on my computer and then change them on my server connected to the Github server
```

Comment:
GitHub is a user interface for accessing git's version control capabilities for our project repositories remotely.